src/components/TextArea.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { TextInputLike } from '../components/Common.js'
+
+const TextAreaContainer = styled.div`
+  margin: 1em;
+`
+
+const TextAreaBox = styled.textarea.attrs({
+  type: 'text',
+})`
+  height: 200px;
+  width: 600px;
+  ${TextInputLike};
+  ::-webkit-scrollbar {
+    width: 10px;
+    height: 10px;
+  }
+  ::-webkit-scrollbar-thumb {
+    background-color: transparent;
+    border-radius: 10px;
+    border: 1px solid ${p => p.theme.colors.highlight};
+  }
+  ::-webkit-scrollbar-track {
+    background-color: transparent;
+  }
+  ::-webkit-scrollbar-corner {
+    background-color: transparent;
+  }
+  ::-webkit-resizer {
+    background-color: transparent;
+  }
+`
+const ErrorMsg = styled.p`
+  color: ${p => p.theme.colors.warning};
+  margin: 7px 0px;
```

Comment:
`em`?

Reply:
oh sure (i got the px from figma mb)

Reply:
![image](https://user-images.githubusercontent.com/47487758/99894801-7b676b80-2c3b-11eb-9cfc-a3fc1feac8ba.png)